<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="result_show">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border-bottom: 1px solid gray;
}

background-color : white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(154, 205, 50);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #6B8E23, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(154, 205, 50);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #6B8E23, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(154, 205, 50);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #6B8E23, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(154, 205, 50);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #6B8E23, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(154, 205, 50);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #6B8E23, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_right">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(135, 206, 235);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_left">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(135, 206, 235);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_point">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(154, 205, 50);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #6B8E23, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(154, 205, 50);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #6B8E23, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(154, 205, 50);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #6B8E23, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(154, 205, 50);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #6B8E23, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(154, 205, 50);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #6B8E23, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_res">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_del">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plot">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(255,20,147);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF1493, stop: 1 #C71585);
}</string>
    </property>
    <property name="text">
     <string>PLOT</string>
    </property>
   </widget>
   <widget class="QLabel" name="max_x">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string> max X =</string>
    </property>
   </widget>
   <widget class="QLabel" name="max_y">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string> max Y =</string>
    </property>
   </widget>
   <widget class="QLabel" name="min_y">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string> min Y =</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clean_plot">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(135, 206, 235);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>CLEAN </string>
    </property>
   </widget>
   <widget class="QLabel" name="min_x">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string> min X =</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="min_x_value">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>-10</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="max_x_value">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="min_y_value">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>-10</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="max_y_value">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="x_">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>         X =</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="x_value">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>491</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Кредитный калькулятор</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
